dist/s_all

```diff
@@ -91,7 +91,7 @@ run "python test_tag.py"
 # The s_mentions script requires bash.
 run "./s_mentions" "--warning-only"
 if command -v evergreen > /dev/null; then
-	run "evergreen validate ../test/evergreen.yml"
+	run "evergreen validate -p wiredtiger ../test/evergreen.yml"
```

Comment:
This is not in scope of WT-9388, but creating a ticket for it seems like overkill.
This stops the 
```
####################### MESSAGE ############################
s_all run of: "evergreen validate ../test/evergreen.yml" resulted in:
    WARNING: no project specified; validation will proceed without checking project settings
    ../test/evergreen.yml is valid with warnings
#######################
```
warning when calling `s_all`